src/Lykke.Service.BlockchainWallets/Controllers/WalletsController.cs

```diff
@@ -123,6 +123,15 @@ public async Task<IActionResult> GetAddress([FromRoute] string blockchainType, [
             if (!ValidateRequest(blockchainType, ref assetId, clientId, out var badRequest))
                 return badRequest;
 
+            if (!_blockchainIntegrationService.BlockchainIsSupported(blockchainType))
```

Comment:
Consider moving this and other common checks into the `ValidateRequest` method. The method itself should return string (the first encountered problem explanation) or null. Smthng like this:
```
var result = ValidateRequest(...);
if (!string.IsNullOrWhiteSpace(result))
    return BlockchainWalletsErrorResponse.Create(result);
```